release-tools/mktarball.sh

```diff
@@ -10,7 +10,7 @@ RELEASEDATE=`date "+%B %d, %Y"`
 
 # Copy bits in
 mkdir ${DESTDIR} ${DESTDIR}/autotools
-cp scrypt_platform.h main.c FORMAT COPYRIGHT ${DESTDIR}
+cp scrypt_platform.h main.c FORMAT COPYRIGHT get-version.sh ${DESTDIR}
```

Comment:
Without `get-version.sh`, autotools bails because it's called by `configure.ac`.  But you're right that I forgot that we don't distribute autotools.

I've updated the PR to delete it from `DESTDIR` along with other autotools stuff.  If you like it, I'll merge it with the original commit and rewrite history.